### PR TITLE
Color Mixer

### LINKED ISSUE #226: Color Mixer
**Color Mixer using React**

**The Color Mixer is a user-friendly web application that allows you to blend two colors seamlessly. Its sleek design features a large square display for the resulting color, and a lower section for selecting and mixing two colors with a mix button, providing an intuitive and visually engaging experience.**

### PR DESCRIPTION
**Pull request by Sumitwarrior7**
# Title and Issue number 

Title :Color Mixer

Issue No. :#226

Code Stack : React

Close #226

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: react/src/machine-coding/color-mixer/index.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import './styles.module.css';
```
Comment (sadanandpai) [inline react/src/machine-coding/color-mixer/index.jsx:2]: only scoped imports are allowed. Follow the other challenges

File: react/src/machine-coding/color-mixer/index.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    const r2 = parseInt(color2.slice(1, 3), 16);
+    const g2 = parseInt(color2.slice(3, 5), 16);
+    const b2 = parseInt(color2.slice(5, 7), 16);
+
+    const mixedColor = `#${Math.round((r1 + r2) / 2).toString(16)}${Math.round((g1 + g2) / 2).toString(16)}${Math.round(
+      (b1 + b2) / 2
+    ).toString(16)}`;
+
+    return mixedColor;
+  };
```
Comment (sadanandpai) [inline react/src/machine-coding/color-mixer/index.jsx:23]: why can't move this function out of the component by passing the colors as arguments

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by Sumitwarrior7** [comment]
The changes you have specified are done

**Comment by Sumitwarrior7** [comment]
@sadanandpai please review the changes
i have some more prs waiting for this project

**Review by sadanandpai — APPROVED** [review]

**Comment by sadanandpai** [comment]
Please fix the conflicts

**Comment by Sumitwarrior7** [comment]
@sadanandpai Conflicts fixed

**Pull request merged**